ci/container/hang_webserver.py

```diff
@@ -11,36 +11,41 @@ class HTTPRequestHandler(BaseHTTPRequestHandler):
     # counts specific calls to change behaviour after some calls
     counter = 0
 
-    def __respond(self, http_code, content_type='text/plain', body=None, ):
-        if body:
-            self.send_response(http_code, body)
+    def __respond(self, http_code, content_type='text/plain', message=None, body=None, ):
+        if message:
```

Comment:
why do we need this change?

Reply:
Because there was no option to return the actual body from our hang_webserver. What was previously called 'body' was actually returned as message in request with empty body.

To test proper parsing of a request body we need it to contain some value.